cas-server-core/src/main/java/org/jasig/cas/logout/LogoutManagerImpl.java

```diff
@@ -118,15 +118,7 @@ public void setIssueAsynchronousCallbacks(final boolean asyncCallbacks) {
      */
     @Override
     public List<LogoutRequest> performLogout(final TicketGrantingTicket ticket) {
-        final Map<String, Service> services;
```

Comment:
Yes, I wanted to make it work the same for each registry type: memory, jpa, hazelcast etc.